templates/webapi/main/parent_group_overview.html.ep

```diff
@@ -21,6 +21,15 @@
     Last Builds for <%= $group->{name} %>
 </h2>
 
+<p>
+  <a href="<%= url_for('tests_overview')->query(todo => 1, map { (groupid => $_->{id}) } @$children);%>"
```

Comment:
```suggestion
  <a href="<%= url_for('tests_overview')->query(todo => 1, map { (groupid => $_->{id}) } @$children); %>"
```
(The space after the `;` is still missing. Note that tidy doesn't run on templates so you'll have to take care of this yourself.)

Reply:
Thanks, solved